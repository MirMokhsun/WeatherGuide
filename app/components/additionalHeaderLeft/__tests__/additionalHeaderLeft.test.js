import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import AdditionalHeaderLeft from '../';

describe('AdditionalHeaderLeft', () => {
  test('AdditionalHeaderLeft snapshot', () => {
    const renderer = new ShallowRenderer();
    const snapshot = renderer.render(<AdditionalHeaderLeft />);
    expect(snapshot).toMatchSnapshot();
  });
});
