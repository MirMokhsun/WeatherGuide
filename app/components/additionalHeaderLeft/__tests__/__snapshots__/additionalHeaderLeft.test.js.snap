// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdditionalHeaderLeft AdditionalHeaderLeft snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
  style={
    Object {
      "paddingLeft": 10,
    }
  }
>
  <Icon
    allowFontScaling={false}
    color="#fff"
    name="left"
    size={30}
  />
</TouchableOpacity>
`;
